2019-05-Berlin/agenda.md

```diff
@@ -10,43 +10,55 @@ It would be fantastic if each one of the breakouts is facilitated by a person wh
 
 ✨ We're excited for Node.js' 5th round of collaboration IRL! Even better, we're excited that folks from the JSF will be joining us for the first time! ✨
 
-> __Location TBA__,
-(Street Address)
-Berlin, Germany
+##Courtyard by Marriot, Berlin City Center
+Axel-Springer-Str. 55
+10117 Berlin, Germany
 
 
 ## Collaborator Summit Day One: 30th May, 2019
 
-(Time window)
+08:00 - 18:00
```

Comment:
Not opposed, but I think we usually started summits at 9am which might be a bit more convenient on the first day.

Reply:
Sure!  I can change that - I just have it there because we have the room starting at 8 a.m.